test/helpers.coffee

```diff
@@ -123,3 +123,11 @@ test "the `baseFileName` helper returns the file name to write to", ->
     name = baseFileName sourceFileName, yes
     filename = name + ext
     eq filename, expectedFileName
+
+test "the `updateSyntaxError` helper allows `throw null`", ->
+    try
+        error = updateSyntaxError null
+    catch err
+        error = err
+        console.error err.stack
```

Comment:
Just added a test to ensure the updateSyntaxHelper does not block null throwing in the future